yarn-project/pxe/src/contract_function_simulator/oracle/private_execution.test.ts

```diff
@@ -1153,9 +1171,6 @@ describe('Private Execution test suite', () => {
   describe('Historical header in private context', () => {
```

Comment:
```suggestion
  describe('Anchor block header in private context', () => {
```

unrelated - the naming here was stale (we use historical header before Mike came with the cute "anchor block" name)